src/pages/BuySell/BuySell.tsx

```diff
@@ -0,0 +1,220 @@
+import { ChevronRightIcon } from '@chakra-ui/icons'
+import { Avatar, Box, Button, Flex, useColorModeValue, useMediaQuery } from '@chakra-ui/react'
+import { getConfig } from 'config'
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import { Redirect } from 'react-router-dom'
+import gemlogo from 'assets/gem-mark.png'
+import onjunologo from 'assets/onjuno.png'
+import { Card } from 'components/Card/Card'
+import { Page } from 'components/Layout/Page'
+import { Text } from 'components/Text'
+
+export const BuySell = () => {
+  const [supportedCoins, setSupportedCoins] = useState<any[]>([])
+  const [supportedSellCoins, setSupportedSellCoins] = useState<any[]>([])
+  const [redirectToGem, setRedirectToGem] = useState(false)
+
+  const bgGray = useColorModeValue('gray.200', 'gray.700')
+  const bgRed = useColorModeValue('red.100', 'red.800')
+  const bgGreen = useColorModeValue('green.100', 'green.800')
+  const [isLargerThan550] = useMediaQuery('(min-width: 550px)')
+
+  const getCoinifySupportedCurrencies = async () => {
+    try {
+      let coinifyResponse = await (
```

Comment:
please do this across two lines rather than jested awaits, just return the last one, don't let and assign